### ISSUE TITLE
describe new websocket protocol?

### ISSUE THREAD

**Issue by benoitc** [aeternity/protocol#234]
it seems that the websocket protocol described here is legacy.  the code in epoch: 
https://github.com/aeternity/epoch/blob/master/apps/aehttp/src/sc_ws_handler.er is using json rpc 2.o apparently.l

**Comment by uwiger** [comment]
Yes, this will be my next task, now that there is a passing test suite for both legacy and JSON-RPC.

**Comment by hanssv** [comment]
For reference the referenced issue is bogus and can be ignored...
